modules/cluster-logging-collector-rsyslog.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-rsyslog_{context}"]
+= Configuring the Fluentd remose-syslog plug-in to send logs to an external log aggregator
+
+You can configure Fluentd to send a copy of its logs to an external Rsyslog server, 
+and not the default Elasticsearch, using the *fluent-plugin-remote-syslog* plug-in.
```

Comment:
See above - this is not specific to rsyslog - rsyslog is just one product out of many that supports the syslog protocol.